snakemake_executor_plugin_slurm/submit_string.py

```diff
@@ -0,0 +1,67 @@
+from snakemake_executor_plugin_slurm_jobstep import get_cpu_setting
+
+
+def get_submit_command(job, params):
+    """
+    Return the submit command for the job.
+    """
+    call = (
+        f"sbatch "
+        f"--parsable "
+        f"--job-name {params['run_uuid']} "
+        f"--output \"{params['slurm_logfile']}\" "
+        f"--export=ALL "
+        f"--comment \"{params['comment_str']}\""
+    )
+
+    # check whether an account is set
+    if params.get("account"):
+        call += f" {params['account']} "
+    # check whether a partition is set
+    if params.get("partition"):
+        call += f" {params['partition']}"
```

Comment:
Just for the sake of consistency :smile: 

```suggestion
        call += f" {params.account} "
    # check whether a partition is set
    if params.get("partition"):
        call += f" {params.partition}"
```

Reply:
I am still on it. Should not requested your input, yet. ;-)

Reply:
done. Needed to change the access pattern.